### PR TITLE
fix(data): database update

### LINKED ISSUE #1661: [Bug] Profile page for MLCommons does not show any challenge cards
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What projects are you seeing the problem on?

OpenChallenges

### Current behavior

The metadata for MLCommons shows 1 challenge, but in the Challenges tab, the results are listed as 0 (and subsequently, no challenge cards):

Screen Shot 2023-06-19 at 2 18 36 PM

### Expected behavior

The paginator should indicate a result of 1, as well as display one card.

### Anything else?

_No response_

### Commit ID

_No response_

### Are you developing inside the dev container?

- [X] I am using the dev container

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by vpchung**
Fixes #1661 

## Changelog

* `challenge_contribution` tables updated in the challenge and organization services (651 rows → 693)

## Preview

**Database**

Screen Shot 2023-06-19 at 4 14 46 PM

**App**

Screen Shot 2023-06-19 at 4 11 49 PM

### CODE REVIEW

**Review by tschaffter — APPROVED** [review]

**Review by tschaffter — CHANGES_REQUESTED** [review]
There are more changes than listed under Changelog. What are the other changes?

File: apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql [hunk 1]
```diff
@@ -2468,645 +2468,687 @@ VALUES ('1', 1, 75, 'sponsor'),
...
-  ('643', 12, 1, 'sponsor'),
-  ('644', 10, 1, 'sponsor'),
-  ('645', 9, 1, 'sponsor'),
-  ('646', 6, 1, 'sponsor'),
-  ('647', 7, 1, 'sponsor'),
-  ('648', 5, 1, 'sponsor'),
-  ('649', 4, 1, 'sponsor'),
-  ('650', 1, 1, 'sponsor'),
-  ('651', 3, 1, 'sponsor');
+  ('13', 5, 134, 'challenge_organizer'),
```
Comment (tschaffter) [inline apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql:2471]: The property `id` of the table `challenge_contribution` is defined as an int, but here data set the ids as string.

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql (hunk 1 above)
Comment (vpchung) [reply apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql:2471]: I thought that was weird too, but I just followed suit what was [done previously](https://github.com/Sage-Bionetworks/sage-monorepo/blob/027f56280792878183549532dd55d2a13bcbf69d/apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.1.0__insert_data.sql).

File: apps/openchallenges/organization-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql [hunk 2]
```diff
@@ -2701,696 +2701,696 @@ Prize4Life seeks to create breakthroughs in effective treatments for Amyotrophic
 -- contributor_roles data
 
 INSERT INTO challenge_contribution (id, challenge_id, organization_id, role)
-VALUES (1, 1, 75, 'sponsor'),
```
[the remark this answers is not in the document]
Comment (vpchung) [reply apps/openchallenges/organization-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql:2704]: Okay, will update.

**Comment by vpchung** [comment]
> What are the other changes?

@tschaffter oh, additional annotations were added to some challenges.  In a previous effort (I think when we were implementing challenge count into the UI?), I noticed that some organizations had a count of 0.  So I went back and tracked down what challenges they were associated with.

**Pull request merged**